include/dmtcp.h

```diff
@@ -281,41 +285,6 @@ int dmtcp_enable_ckpt(void) __attribute__((weak));
 #define dmtcp_enable_ckpt() \
   (dmtcp_enable_ckpt ? dmtcp_enable_ckpt() : DMTCP_NOT_PRESENT)
 
-/**
- * Example:  dmtcp_get_libc_addr("fork") returns the address of "fork"
- *           in libc at runtime (when loaded in memory).
- * NOTE:  dmtcp_get_libc_addr(fnc) skips any DMTCP wrapper functions around
- *        fnc, and directly calls its definition in libc.  In contrast, the
- *        _real_XX() functions found in DMTCP_ROOT/src are pointers to the
- *        next definition of fnc in library search order.  (This may be in the
- *        next libdmctp_YY.so library in search order, or in libc.so istelf.)
- * EXAMPLE USAGE 1:
- *   static typeof(fork) *libc_fork_ptr = NULL;
- *   if (libc_fork_ptr == NULL) {libc_fork_ptr = dmtcp_get_libc_addr("fork");}
- *   if (libc_fork_ptr != DMTCP_NOT_PRESENT) {
- *     // Skip DMTCP interposition on fork; child doesn't coonect to coord.
- *     (*libc_fork_ptr)();
- *   }
- * EXAMPLE USAGE 2:
- *   static typeof(execvp) *libc_execvp_ptr = NULL;
- *   if (libc_execvp_ptr == NULL) {
- *     libc_execvp_ptr = dmtcp_get_libc_addr("execvp");
- *   }
- *   if (libc_execvp_ptr != DMTCP_NOT_PRESENT) {
- *     // Don't preload DMTCP libraries when you exec to a new program.
- *     char *ld_preload = getenv("LD_PRELOAD");
- *     ld_preload[0] = '\0';
- *     (*libc_execvp_ptr)(...);
- *   }
- */
-void * dmtcp_get_libc_addr(const char* libc_fnc) __attribute__((weak));
```

Comment:
Do we not need `dmtcp_get_libc_addr` function at all?
If we have to, I'd suggest either putting it under `#if 0 ...  #endif` or removing the remaining references (in `src/execwrappers.cpp` and `src/dmtcpnohijackstubs.cpp`) in DMTCP. What do you think @gc00?

Reply:
I think we still need dmtcp_get_libc_addr.  We're not actively using it right now.  But we will need it in a scenario where several plugins (or a DMTCP plugin and the target application) are interposing on a call in glibc.  At some point, one of these plugins will want to directly call the glibc low-level function, without fear of the target application or another plugin interposing on it.
@karya0 ,  If you don't want it to interfere for now, then please enclose it in `#if 0 ... #endif`.  But this should not be dead code.